packages/api-docs/src/tableRowJson.ts

```diff
@@ -29,6 +29,7 @@ export function createTableRowJson(collectedData: ICollectedData, apiItem: ApiIt
         apiProperty.excerptTokens,
         apiProperty.propertyTypeExcerpt.tokenRange
       );
+      break;
```

Comment:
This seems like a function that would benefit from some tests. Was the lack of `break;` causing issues, or just here for consistency?

Reply:
It was making properties get parsed/output as methods. 🤦‍♀ It would be nice if there was a lint rule for "no switch case fall-through except on consecutive lines"...

Reply:
While I totally agree it would be good to have tests, that would require more time/thought to set up due to the reliance on API Extractor models. So I'd rather not do that right now.